orchagent/orchdaemon.cpp

```diff
@@ -186,6 +186,8 @@ bool OrchDaemon::init()
     gDirectory.set(vnet_rt_orch);
     VRFOrch *vrf_orch = new VRFOrch(m_applDb, APP_VRF_TABLE_NAME, m_stateDb, STATE_VRF_OBJECT_TABLE_NAME);
     gDirectory.set(vrf_orch);
+    VrrpOrch *vrrp_orch = new VrrpOrch(m_applDb, APP_VRRP_TABLE_NAME);
```

Comment:
APP_VRRP_TABLE_NAME name is not defined. It needs to be defined in common/schema.h.

Reply:
Will add APP_VRRP_TABLE_NAME in common/schema.h.

Reply:
https://github.com/sonic-net/sonic-swss-common/pull/927

Reply:
Philo has already added it.
https://github.com/sonic-net/sonic-swss-common/pull/813